content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+**Note:** This documentation describes how to configure CI to assist with Bugzilla-related processes. It does _not_ describe
+how these processes look. Documentation about the Openshift Bugzilla process can be found [here][6].
+
+## Linking Pull Requests to Bugzilla bugs
+
+Pull requests can reference a Bugzilla bug in their title, for example
+`Bug 123: Rest of the PR title`. A PR can at most reference one Bugzilla bug. Referencing a bug makes the bugzilla plugin run a couple
+of checks:
+* If a bug with the id `123` exists
+* If repo has the [`is_open: true`][3] setting, the bug must not be closed
+* If the target release of the bugzilla bug matches the configured target release for the branch the PR points at. This mapping is
+  configured in the openshift/release repository via the `target_release` property. [Here][2] for example the `release-4.6` branch of the
```

Comment:
However, having a section at the bottom of the doc with a reference snippet of configuration (or two) that show off how to configure the plugin and use effective overrides may be helpful. These details belong there.